### PR TITLE
[ADD] #3631, Restrictions on create and edit on different views

### PR DESCRIPTION
**Pull request by wytzejan**

### CODE REVIEW

File: bbc_restrictions/views/pricelist_view.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+		<record model="ir.ui.view" id="bbc_restrictions_product_pricelist_version_form_view">
+			<field name="name">product.pricelist.version.form.restriction.inherit</field>
```
Comment (StefanRijnhart) [inline bbc_restrictions/views/pricelist_view.xml]: Looks like you mix tabs and spaces! Can you set your editor to use 4 spaces (not a tab) and reformat?

Comment (wytzejan) [reply bbc_restrictions/views/pricelist_view.xml]: You are right! My latest commit fixed the issue.

File: bbc_restrictions/views/mrp_view.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        		<xpath expr="/form/group/group/field[@name='product_id']" position="attributes">
+        			<attribute name="options">{'no_create': true}</attribute>
+        		</xpath>
+        		<xpath expr="/form/group/group/field[@name='product_tmpl_id']" position="attributes">
+        			<attribute name="options">{'no_create': true}</attribute>
+        		</xpath>
+                <xpath expr="/form/group/group/div/field[@name='product_uom']" position="attributes">
+                    <attribute name="options">{'no_create': true}</attribute>
+                </xpath>
+        		<xpath expr="/form/notebook/page[@string='Components']//field[@name='product_id']" position="attributes">
```
Comment (StefanRijnhart) [inline bbc_restrictions/views/mrp_view.xml]: The indentation is still off

Comment (wytzejan) [reply bbc_restrictions/views/mrp_view.xml]: You're right! I was too soon with my commit and did not check the other views.
I installed a tool in my editor now that checks the XML indentation for each XML-file automatically. I re-indent every XML-file with this tool.
What you'll see is that all XML code is displayed without white lines. Do you prefer these white lines for easy reading? Or is the XML code OK as it is in my last commit?

File: bbc_stock/views/product.xml [hunk 3]
```diff
@@ -1,59 +1,78 @@
...
+                    <attribute name="options">{'no_create': True}</attribute>
+                </field>
+                <field name="supplier_taxes_id" position="attributes">
+                    <!-- Supplier Taxes -->
                     <attribute name="options">{'no_create': True}</attribute>
                 </field>
                 <field name="alternative_product_ids" position="attributes">
                     <attribute name="options">{'no_create': True}</attribute>
-                    <attribute name="domain">"[('is_synced_magento', '=', True)]"</attribute>
+                    <attribute name="domain">&quot;[('is_synced_magento', '=', True)]&quot;</attribute>
```
Comment (StefanRijnhart) [inline bbc_stock/views/product.xml]: Your formatting tool exposed this anomality. Can you remove the inner quotes?

**Comment by StefanRijnhart** [comment]
BTW I usually hesitate to use formatting tools because it can easily lead to conflicts especially on collaborative projects but we don't have a lot of unmerged code at the moment, I guess.

File: bbc_restrictions/views/mrp_view.xml (hunk 2 above)
Comment (StefanRijnhart) [reply bbc_restrictions/views/mrp_view.xml]: Not my preference, but it's fine really. If you remove the \&quot; in the domains we can proceed.

Comment (wytzejan) [reply bbc_restrictions/views/mrp_view.xml]: I agree. I refactored the xml code to what it was.
&quot; has been removed from the code.

**Comment by wytzejan** [comment]
I think we can proceed now but I'd like to wait until the other PR is ready to proceed. Otherwise, we will have the same error in production as we had before. I will look into the other PR as soon as possible.

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**